installing/installing_sno/install-sno-installing-sno.adoc

```diff
@@ -139,7 +139,11 @@ include::modules/creating-custom-live-rhcos-iso.adoc[leveloffset=+1]
 [id="install-sno-with-ibmz"]
 == Installing {sno} with {ibm-z-title} and {ibm-linuxone-title}
 
-Installing a single-node cluster on {ibm-z-name} and {ibm-linuxone-name} requires user-provisioned installation using either the "Installing a cluster with {op-system-base} KVM on {ibm-z-name} and {ibm-linuxone-name}" or the "Installing a cluster with z/VM on {ibm-z-name} and {ibm-linuxone-name}" procedure.
+Installing a single-node cluster on {ibm-z-name} and {ibm-linuxone-name} requires user-provisioned installation using one of the following procedures:
+
+* Installing a cluster with z/VM on {ibm-z-name} and {ibm-linuxone-name}
+* Installing a cluster with {op-system-base} KVM on {ibm-z-name} and {ibm-linuxone-name}
+* Installing a cluster in an LPAR on {ibm-z-name} and {ibm-linuxone-name}
```

Comment:
Since this is an assembly, wdyt about making these into xrefs so the user can just click what they want?